CHANGELOG.md

```diff
@@ -4,51 +4,6 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-[3.6.0] - 2021-12-03
+[3.6.2] - 2021-12-07
 ### Changed
-- Azure DevOps: The state of deleted scenarios from the feature files will be changed to closed.
```

Comment:
Should these lines be removed?

Reply:
I've removed the lines because release notes become very long.
![image](https://user-images.githubusercontent.com/24970976/145157976-eb1eae0f-a8fa-4448-953a-97688694804e.png)